plugin-core/src/main/java/appland/settings/AppMapApplicationSettings.java

```diff
@@ -31,6 +33,13 @@ public class AppMapApplicationSettings {
      */
     private volatile boolean showFirstAppMapNotification = false;
 
+    /**
+     * Map of environment variables to be set when starting AppMap services
+     * The key is the name of the environment variable and the value is the value of the environment variable.
+     */
+    private volatile Map<String, String> cliEnvironment = new HashMap<>();
```

Comment:
nit: Oh, sorry, I missed this in the first pass. You could make this `Map.of()` to set an empty, unmodifiable map.